demos/coreMQTT/mqtt_demo_serializer.c

```diff
@@ -541,39 +544,34 @@ int RunCoreMqttSerializerDemo( bool awsIotMqttMode,
 }
 /*-----------------------------------------------------------*/
 
-static void prvGracefulShutDown( Socket_t xSocket )
+static BaseType_t prvGracefulShutDown( Socket_t xSocket )
 {
-    uint8_t ucDummy[ 20 ];
-    const TickType_t xShortDelay = pdMS_TO_TICKS( 250 );
-    BaseType_t xShutdownLoopCount = 0;
+    BaseType_t xStatus = pdFAIL;
+    int32_t lSocketStatus = ( int32_t ) SOCKETS_ERROR_NONE;
```

Comment:
Since `xStatus` is initialized to `pdFAIL`, should `lSocketStatus` also be initialized to the failure equivalent?

Reply:
Good suggestion. Thanks